### ISSUE TITLE
Images saved via airdrop are not accessible 

### ISSUE THREAD

**Issue by dragonfly91** [GospelAid/badiup#65]
We need the ability to add all images - including those obtained from other apps.

**Comment by dragonfly91** [comment]
Need to check if this issue persists on iPhone X.

**Comment by dragonfly91** [comment]
Need to buy an adapter for USB C to iPhone

**Comment by dragonfly91** [comment]
Can’t test on iOS 13.3.1 due to https://github.com/flutter/flutter/issues/49504 
Need to try downgrading

**Comment by dragonfly91** [comment]
Since this issue can be solved by testing on a downgraded iOS phone, unblocking this.

**Comment by dragonfly91** [comment]
Marking this as wontfix since there is an other way to upload pictures. We can download desired pics from Google Drive app and they will be saved to Camera Roll and are available to the Badiup app.
